python/tk_multi_data_validation/dialog.py

```diff
@@ -114,8 +114,8 @@ def _setup_ui(self):
             lambda rules: self._manager.validate()
         )
         self._validation_widget.fix_rules_callback = self._resolve_rules
-        self._validation_widget.fix_all_callback = lambda rules: self._manager.resolve(
-            pre_validate=True, post_validate=True
+        self._validation_widget.fix_all_callback = (
+            lambda rules: self._resolve_all_rules()
```

Comment:
Call the hook method (that replaces these functions).

For fix all, pass the set of rules to the hook even though the manager does not use them. This way we can still check if post fix refresh is needed, also in the future it might be useful to have them.